omisego-sdk/src/main/java/co/omisego/omisego/operation/Listenable.kt

```diff
@@ -44,9 +43,10 @@ interface Listenable {
 fun TransactionRequest.startListeningEvents(
     client: SocketClientContract.Client,
     payload: Map<String, Any> = mapOf(),
-    listener: TransactionRequestListener
+    listener: SocketCustomEventListener.TransactionRequestListener
 ) {
     with(client) {
+        listener.strategy = FilterStrategy.Topic(socketTopic)
```

Comment:
As I said, this can be dangerous, since you'll change the behavior of a given object, the dev might not expect this.
Instead, I think we should wrap the given listener in a listener with the given strategy.
Maybe we should even set the `strategy` in the listener as final?